www/docs/guide/wiring.md

```diff
@@ -56,6 +56,24 @@ Other tested adaptors
 
    Requires `READ_SENSORS_BATCH_SIZE` set to 8 or less
 
+   A 120 Ohm (brown-red-brown) resistor may be required for data communication between the adapter and inveter to occur without corruption. The resistor should be added between the A+B lines, such as in the example below.
+
+   > NOTE: The resistor legs should be trimmed to a more reasonable length to ensure they don't accidentally short together.
```

Comment:
You should trim the one in the picture 😆 

Can you add size info to the image tags? It seems `=400x400` is quite common on this page, but maybe the resistor should be `=200x200`?

Reply:
Haha yeah, I've been meaning to trim it, it was temporary, but you know how it is; as soon as it was functional, I now don't want the downtime.

I'll try and find a moment to get a multimeter on it tomorrow and get the voltage drop.

200x200 seems right for this image, I'll add the size info (I tried it originally before submitting but it broke the rendering in my IDE so I removed it thinking perhaps it was for specific types).

Reply:
The size is not 100% standard markdown. The docs use a plugin to enable sizing

Thanks!